README.md

```diff
@@ -101,6 +101,14 @@ are not the same AND the tablespace directories do not already exist, then
 manual intervention is necessary for creating all the tablespaces' directories
 and assigning them the correct owner.
 
+### Oracle-specific notes
+
+This install script assumes SYSDBA privileges in order to create the users that the application will use. 
+
+You will need to create the tablespaces `TRANSMART` and `INDX` (sic) manually before creating the database. The install script can do this automatically if you set `ORACLE_MANAGE_TABLESPACES=1`, but in that case you will need to set `ORACLE_TABLESPACES_DIR` to a directory name (on the database server) where Oracle can automatically create the files to store the new tablespaces and other database objects.
```

Comment:
I see now that not setting `ORACLE_TABLESPACES_DIR` doesn't do any good because  transmart-data will refuse to create tablespaces without it set. This should be fixed though.